lib/generators/atomic_cms/scaffold/scaffold_generator.rb

```diff
@@ -0,0 +1,68 @@
+require 'active_support/core_ext/string'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.